src/app/api/[[...route]]/route.ts

```diff
@@ -10,45 +10,48 @@ import categoryRoute from './routes/category.route';
 import taskRoute from './routes/task.route';
 import tenantRoute from './routes/tenant.route';
 
-const app = new Hono().basePath('/api');
-
-const defaultRoutes = [
-  { path: '/sharehouse', route: sharehouseRoute },
-  { path: '/sharehouses', route: sharehousesRoute },
-  { path: '/rotation', route: rotationRoute },
-  { path: '/category', route: categoryRoute },
-  { path: '/task', route: taskRoute },
-  { path: '/tenant', route: tenantRoute },
-];
-
-/**
- * This is a test route to check if the user is logged in.
- * TODO: Remove this route before deploying to production.
- */
-app.get('/whoami', async (c) => {
-  const session = await auth();
+const app = new Hono();
+
+const routes = app
+  .basePath('/api')
+  .route('/sharehouse', sharehouseRoute)
+  .route('/sharehouses', sharehousesRoute)
+  .route('/rotation', rotationRoute)
+  .route('/category', categoryRoute)
+  .route('/task', taskRoute)
+  .route('/tenant', tenantRoute)
+
+  /**
+   * This is a test route to check if the user is logged in.
+   * TODO: Remove this route before deploying to production.
+   */
+  .get('/whoami', async (c) => {
+    const session = await auth();
+
+    if (!session) {
+      return c.json(
+        {
+          error: 'You are not logged in!',
+        },
+        401,
+      );
+    }
+
+    const landlord = await prisma.landlord.findUnique({
+      where: { id: session.user.id },
+    });
 
-  if (!session) {
     return c.json({
-      message: 'You are not logged in!',
+      message: 'You are logged in!',
```

Comment:
Could you change the `message` to the `error` for the type safety?

Reply:
Hi. Thank you. I've fixed it.